.gitignore

```diff
@@ -40,3 +40,5 @@ testem.log
 #System Files
 .DS_Store
 Thumbs.db
+
+*node_modules
```

Comment:
It looks like `node_modules` is already captured further above in the `.gitignore` file.